src/less/date-textbox/date-textbox.less

```diff
@@ -0,0 +1,29 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.date-textbox {
+    // 343px - (2 * @spacing-200) [not autmatically compiled by LESS]
+    --calendar-month-width: 311px;
+}
+
+.date-textbox__popover {
+    .background-color-token(calendar-background-color, color-background-primary);
```

Comment:
@agliga  and I talked about this in person, since this is also a problem with all components that use bubble mixins I am going to update everything in a follow-up PR.